.circleci/config.yml

```diff
@@ -0,0 +1,112 @@
+version: 2.1
+
+executors:
+  my-executor:
+    docker:
+      - image: continuumio/miniconda3
+      # astroconda/python:3.6.1
```

Comment:
Might be best to say something like "astroconda is installed below"?  Or were you thinking to switch over to a specific docker container down the line?